crypto/engine/eng_cryptodev.c

```diff
@@ -184,7 +186,6 @@ static struct {
     {
         CRYPTO_AES_CBC, NID_aes_256_cbc, 16, 32,
     },
-# ifdef CRYPTO_AES_CTR
```

Comment:
Quite right.  If it were me, I would restore like this:

```
# if !defined(CRYPTO_ALGORITHM_MIN) || defined(CRYPTO_AES_CTR)
```

`CRYPTO_ALGORITHM_MIN` is a macro that I've observed on FreeBSD that cryptodev-linux doesn't define.  You might want to add a comment to this effect somewhere.